README.md

```diff
@@ -1 +1,26 @@
-# halcyonsim
\ No newline at end of file
+# HalcyonSim
+
+Simulates a virtual world environment. With sun, moon, clouds, land, buildings *(objects with seamless textures to give the desired look)*, avatars.. and interactive objects that are scripted.
+
+HalcyonGrid or HalcyonSim was a fork from the <a href="http://opensimulator.org" target="_blank" rel="nofollow" >OpenSimulator</a> Project in 2010, 
```

Comment:
I'd prefer to use the normal markdown link format. When I started web development I used to use `target="_blank"` a lot, but since then I've realized that as a user it's my choice to open a new tab or not, not the author of the page's choice how I navigate.

Reply:
The normal markdown link format works, aye, though the `rel` tag can't be used on the markdown link format, and have removed the `target="_blank"` in the anchors, that's a reasonable reason to not have them in there. Although on regular websites not through github, I think the `target="_blank"` is useful for links on external websites.

Reply:
> Additionally I'm not sure that this will help the website as I don't think the readme feeds the site. The website code is expected to be created using a different set of tools I haven't completed setting up.

The website [https://halcyonsim.org](https://halcyonsim.org) has at the "bottom"  "This site is open source. [Improve this page](https://github.com/HalcyonSim/halcyonsim/edit/master/README.md)."  (which is pretty much at the top right now, since there's nothing on the page, but the title.)